seed/src/main.rs

```diff
@@ -28,13 +28,13 @@ use argh::FromArgs;
 #[derive(FromArgs)]
 /// Radicle Seed.
 pub struct Options {
-    /// track the specified peers only
+    /// track the specified peer only
     #[argh(option)]
-    pub track_peers: Vec<PeerId>,
+    pub track_peer: Vec<PeerId>,
```

Comment:
Not sure this suggestion works in GitHub, but could you try this so we can get a mix of the correct CLI param name and also the plural for the actual field
```suggestion
#[argh(option, name = "track_peer")]
pub track_peers: Vec<PeerId>
```